wiki/Registration/id.md

```diff
@@ -1,40 +1,51 @@
-<span style="font-size:18px">1.Pergi ke <span class="plainlinks"> [website](https://osu.ppy.sh)</span> dan klik "<b>I'm new!</b>" untuk melanjutkan ke formulir pendaftaran. </span>
+## Bagaimana cara saya mendaftar?
 
-<span style="font-size:18px">2. Isilah formulir di bawah ini kemudian tekan "Continue" </span>
+**Harap diperhatikan bahwa memiliki lebih dari satu akun setiap saat merupakan salah satu pelanggaran terhadap [peraturan osu!](/wiki/Rules/). Ikuti langkah-langkah di bawah ini untuk mendaftarkan akun osu! anda.**
 
-| <b>Legenda</b>                                                                                                                              |
-|---------------------------------------------------------------------------------------------------------------------------------------------|
-| **1. Nama Pengguna**                                                                                                                        |
-| a. Maksimal 15 karakter                                                                                                                     |
-| b. Karakter angka dan huruf ("a-z, A-Z, 0-9"), garis bawah ("\_"), kurung kotak, ("\[" and "\]") dan tanda hubung ("-") yang diperbolehkan. |
-| **2. Alamat E-mail**                                                                                                                        |
-| **3. <b>Kata Sandi</b>**                                                                                                                    |
-| **4. Catatan**                                                                                                                              |
-| **5. Tombol "Continue"**                                                                                                                    |
+1. Di [halaman utama web](https://osu.ppy.sh), klik ``I'm new!`` untuk memulai proses pendaftaran.
+![Tombol registrasi](Register-1.png "Tombol registrasi")
+2. Isi semua kolom dalam formulir di bawah dan tekan ``Continue`` ![Kolom akun](Register-2.png "Kolom akun")
+    1. Nama Pengguna
+        - Jumlah maksimumnya 15 karakter
+        - Karakter alfanumerik (``a-z``, ``A-Z``, ``0-9``), garis bawah (``_``), kurung kotak, (``[`` and ``]``), setrip (``-``), dan spasi (`` ``) merupakan karakter yang diizinkan.
+    2. Alamat E-Mail
+    3. Kata Sandi
+    4. Catatan
+    5. Lanjut
+3. Masuk ke *osu!* dan proses pendaftaran akan selesai. Anda telah berhasil mendaftarkan akun *osu!* anda!
 
-<span style="font-size:18px">3. Masuk ke osu! dan kamu pun siap! </span>
+## Masuk ke osu!
 
-<img src="Register-3.png" title="fig:Register-3.png" alt="Register-3.png" width="200" /><img src="Login.png" title="fig:Login.png" alt="Login.png" width="120" />
+### osu! executable
 
-<img src="Userpanel.png" title="Userpanel.png" alt="Userpanel.png" width="320" />
+![Register.png](Register-3.png)  
```

Comment:
translate the Register.png into something like `osu! executable`

same for images below